src/modules/m_clones.cpp

```diff
@@ -23,16 +23,97 @@
 
 /** Handle /CLONES
  */
```

Comment:
This comment belongs to the CommandClones class, should put the new code above it